handlers/reload.yml

```diff
@@ -0,0 +1,41 @@
+# file: postgresql/handlers/reload.yml
+  
+#
+# Reload PostgreSQL
+#
+- name: PostgreSQL | <= v9.4, must restart to implement parameter changes
+  debug:
+    msg: "NOTICE: Detected PostgreSQL v9.4 or earlier. Restart required."
+  when: postgresql_restart_on_config_change and
+        postgresql_version | version_compare('9.4','<=')
+  register: postgresql_pre95_restart_required
+  
+- name: PostgreSQL | Reload PostgreSQL to implement parameter changes  | >= v9.5
+  service:
```

Comment:
I'm only using `service:` everywhere, because it auto-detects and will handle both service, systemd, plus a bunch of others.  It's "Stable Interface", so it's not going to change in the future... right?

Is there any advantage to splitting out service and systemd into separate calls?

Reply:
@gclough the `service` Ansible module does not seems to "reload" the unit files from disk, resulting in a systemd warning, but for all other operations it's OK. We use this in the role I maintain: https://github.com/OCA/ansible-odoo/blob/master/handlers/main.yml